api/project.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+
+unless Dir.exists? ".project/common"
+  unless system(*%W{mkdir -p .project})
+    STDERR.puts "mkdir failed."
+    exit 1
+  end
+  unless system(*%W{git clone https://github.com/dmohs/project-management.git .project/common})
```

Comment:
We've had some nastiness with git submodules in FireCloud and I think it caused me to steer away. A submodule would be more transparent, so I'll try that out.